internal/pkg/pct/install.go

```diff
@@ -23,8 +36,33 @@ type PctInstallerI interface {
 	Install(templatePkg string, targetDir string, force bool) (string, error)
 }
 
+func init() {
+	Client = &http.Client{}
+}
+
 func (p *PctInstaller) Install(templatePkg string, targetDir string, force bool) (string, error) {
 
+	// If the package path is a URI, download tar to temp folder
+	u, _ := url.ParseRequestURI(templatePkg)
```

Comment:
Do we want to inspect the error back here and/or do some prior checking like a, `strings.HasPrefix(templatePkg, "http")`, as we could be given a dodgy URI and then continue on processing it as if it were a disk resource.

Reply:
I genuinely forgot the `HasPrefix` function existed 😓 I'll rework for this